web/app/themes/justicejobs/search-apply-page.php

```diff
@@ -248,7 +248,12 @@ class="hero__img hero__img--mobile"
                                 </p>
                             </td>
                             <td>
-                                <a href="<?php the_permalink(); ?>" class="btn btn--blue btn--small">View</a>
+                                <?php
+                                $jj_view_label = strstr(get_the_title(), ' &#8211;', true);
```

Comment:
The latest import adds the en dash and Job ID to the end of the job title (not the start) so you should not need to do the strstr.

Reply:
strstr() helps removes the numbers off the end so the screen reader doesn't read them out of context - I mentioned this above.

Reply:
The result is "View Senior Project Manager" instead of "View Senior Project Manager - 30008" - is that OK? @RobLoweMOJ

Reply:
Apologies i didn't notice the true in before needle..thought it was outputting after the needle haha . All good.

Reply:
Thank you 🙏